src/main/java/com/wwi21sebgroup5/cinema/services/MovieService.java

```diff
@@ -111,6 +116,7 @@ public Movie add(MovieRequestObject movieObject)
         return m;
```

Comment:
Baruchen wir hier dann auch noch mehr Attribute? Also Trailer Character für die Schauspieler und so?

Reply:
Trailer werden wir wohl nicht finden können, deswegen ist das nullable. Rating usw. + Charakter für die Schauspieler kann theoretisch noch eingebaut werden in unsere Requests, würde wohl Sinn machen